src/StockportWebapp/TagParsers/AlertsInlineTagParser.cs

```diff
@@ -18,7 +18,7 @@ public string Parse(string content, IEnumerable<Alert> alertsInline)
         foreach (Match match in matches)
         {
             var AlertsInlineTitle = match.Groups[1].Value;
-            var AlertsInline = GetAlertsInlineMatchingTitle(alertsInline, AlertsInlineTitle);
+            var AlertsInline = GetMatchingInlineAlert(alertsInline, AlertsInlineTitle);
```

Comment:
Use explicit type instead of var (started doing this recently to everything we can)